packages/compass-aggregations/src/components/stage-preview/output-stage-preview.tsx

```diff
@@ -85,60 +87,10 @@ export const MergeStage = ({
   onRunOutputStage,
   onGoToOutputResults,
 }: OutputStageProps) => {
-  if (isLoading) {
-    return <Loader destinationNamespace={destinationNamespace} />;
-  }
-
-  // Stage editor show the error message.
-  if (hasServerError) {
+  if (!['$out', '$merge'].includes(operator || '')) {
```

Comment:
We can probably use a helper that we use in other places

```suggestion
  if (isOutputStage(operator ?? '')) {
```